package.json

```diff
@@ -20,7 +21,8 @@
     "start": "node -r dotenv/config index.js",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "tdd": "yarn run test --watch --verbose false",
-    "test": "NODE_ENV=test jest --coverage --config=jest.config.js"
+    "test": "NODE_ENV=test jest --config=jest.config.js",
+    "test:coverage": "NODE_ENV=test jest --coverage --config=jest.config.js"
```

Comment:
i actually dont know if --coverage is required anymore, can you just try dropping `test:coverage`

Reply:
It's required. That flag causes jest to generate the coverage data, which the `codecov` tool will subsequently process and upload. We just don't want to generate that data every time unit tests are run.

Reply:
hmm when i run npm test it always generates a coverage, especially because jest.config has coverage enabled?